lib/ssf/local_buffering_client.rb

```diff
@@ -9,7 +9,7 @@ def initialize(service: '')
     end
 
     def send_to_socket(span)
-      @buffer << span
+      @buffer = @buffer << span
```

Comment:
According to https://ruby-doc.org/core-2.2.0/Array.html#class-Array-label-Adding+Items+to+Arrays doesn't `@buffer << span` append? (Your new test passes even if we leave `@buffer << span`)